code/game/gamemodes/steal_items.dm

```diff
@@ -155,7 +159,33 @@
 /datum/theft_objective/traitor/ablative
 	name = "an ablative armor vest"
 	typepath = /obj/item/clothing/suit/armor/laserproof
-	protected_jobs = list("Head of Security", "Warden")
+	protected_jobs = list("Captain", "Head of Security", "Warden")
+
+/datum/theft_objective/traitor/telebaton
+	name = "a telescopic baton"
+	typepath = /obj/item/weapon/melee/telebaton
+	protected_jobs = list("Captain", "Head of Security")
+
+/datum/theft_objective/traitor/planningframe
+	name = "the law planning frame"
+	typepath = /obj/item/weapon/planning_frame
+	protected_jobs = list("Captain", "Research Director")
+
+/datum/theft_objective/traitor/belt
+	name = "the advanced toolbelt"
```

Comment:
You might want to make this clearer - not everyone will know the CE has a special belt.